src/careamics/lvae_training/dataset/vae_dataset.py

```diff
@@ -1000,221 +1051,10 @@ def __getitem__(
         if self._return_index:
             output.append(index)
 
+        # TODO: Is not int index used anywhere in the training code?
```

Comment:
index is always int.  I donot remember but i think https://github.com/CAREamics/careamics/pull/212/files#diff-1d15d26bff672df4657f5ce5f1d3cc98a71924b4f2636573fe5d92bb066018dfL963 returned np.int64.

Reply:
Is it ok to remove the last lines then? (1058-1060)

        
```
_, grid_size = index
output.append(grid_size)
return tuple(output)
```

Reply:
yes. please remove.